### ISSUE TITLE
Package manager ignores packages with a suffix in the version

### ISSUE THREAD

**Issue by aacuevas** [bonsai-rx/bonsai#808]
[Nuget version numbering](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning) accepts the use of suffix strings for beta or release candidate versioning.

However, packages built with any suffix in the version field are not shown in the package manager when tried to load them from a local repository.  We have not tried with remote repositories.

**Comment by glopesdev** [comment]
@aacuevas packages with suffix strings are considered "prerelease" versions, and are shown in the package manager only if the checkbox "Include prerelease" is checked. The behavior should be the same for either local or remote repositories.

Can you confirm if the checkbox is checked? It can be found to the right of the search bar in the package manager.

**Comment by aacuevas** [comment]
Yes, it works as you say. I did not know there was a specific method to enable prereleases.

However, in my system at least, said checkbox only says "include", making this functionality not obvious.
![image](https://user-images.githubusercontent.com/1830303/129825508-858b1fe9-e5fd-4923-9af9-591a31cc87fa.png)

So knowing this, I should reformulate this as a UI issue. Should I create a new one?

**Comment by glopesdev** [comment]
Thanks, yes, this is a UI issue I've been forgetting to fix. I've created one (#810) for this, do you mind adding the screenshot there?
